### DISCUSSION TITLE
Update fontawesome icon prefix (fal -> fa-light) in icon-set?

### DISCUSSION THREAD

**Discussion by KINGER404** [quasarframework/quasar#13220] [General - Components / Directives / etc] [answered]
Iconset uses old prefix:

https://github.com/quasarframework/quasar/blob/dev/ui/icon-set/fontawesome-v6-pro.mjs

New use:

<img width="809" alt="изображение" src="https://user-images.githubusercontent.com/53098109/164339118-b217b974-b780-41f7-a4aa-910d4a533d39.png">

**Comment by rstoenescu** [comment, marked as answer]
Hi,

Both forms will work. We're just trying to ease our developers into upgrading to FA v6.
